src/app/zap-templates/templates/app/CHIPClientCallbacks-src.zapt

```diff
@@ -226,6 +227,98 @@ void LogStatus(uint8_t status)
     }
 }
 
+void LogIMStatus(uint16_t status)
+{
+    switch (status)
+    {
+    case 0x0: // Success
```

Comment:
Can we directly use the constants from: https://github.com/project-chip/connectedhomeip/blob/1da1a92e5197ad49c1765436881ca6a690d188b6/src/protocols/interaction_model/Constants.h#L66 ?

Reply:
Fixed